CONTRIBUTING.md

```diff
@@ -183,6 +183,20 @@ Here are the trigger phrases for individual checks:
 * `/test-rancher-e2e`: Linux IPv4 e2e tests on Rancher clusters.
 * `/test-rancher-conformance`: Linux IPv4 conformance tests on Rancher clusters.
 * `/test-rancher-networkpolicy`: Linux IPv4 networkpolicy tests on Rancher clusters.
+* `/test-kind-ipv6-e2e`: Kind dual stack e2e tests.
```

Comment:
the pattern is followed to use "_ipv6" in dual stack test trigger phrase , it could be preferred to use "_ds"

Reply:
I used "_ipv6" instead of "_ds"  for dual stack because we have used "_ipv6"  in other non kind dual stack jobs.